draft-ietf-quic-transport.md

```diff
@@ -926,6 +926,23 @@ them with other connection IDs for the same connection.  As a trivial example,
 this means the same connection ID MUST NOT be issued more than once on the same
 connection.
 
+Packets with long headers include at least two connection IDs, which are used to
+determine the connection IDs that are used for the connection when established,
+see {{negotiating-connection-ids}} for details.
+
+Packets with short headers ({{short-header}}) only include the Destination
+Connection ID and omit the explicit length.  The length of the Destination
+Connection ID field is expected to be known to endpoints.  Endpoints that use a
+load balancer that routes based on connection ID could agree with the load
```

Comment:
Run on sentence ("that ... that"). I prefer the earlier construction: "Endpoints using a connection-ID based load balancer could agree with the load	balancer on a fixed or minimum length and on an encoding for connection IDs."

Reply:
It has all the problems already highlighted, plus the precedence of the "or" and "and" toward the end is unclear.  I've tweaked again.